packages/datagateway-common/src/images/datagateway-logo-white.svg

```diff
@@ -0,0 +1,21 @@
+    <path d="M742.64,50.59V72.23h17.5v7.65h-17.5v56.39q0,7,2.54,10.48t8.48,3.45a57,57,0,0,0,7.58-.79l.39,7.59a30.52,30.52,0,0,1-10,1.32q-9.61,0-14-5.59t-4.37-16.44V79.88H717.72V72.23h15.55V50.59Z" transform="translate(-0.5 -0.5)" style="fill: rgb(255, 255, 255);"/>
+    <path d="M809.05,158.32A36.7,36.7,0,0,1,789.56,153,36.2,36.2,0,0,1,776,138.2a46.11,46.11,0,0,1-4.84-21.29v-3.36a49.76,49.76,0,0,1,4.73-22,37.6,37.6,0,0,1,13.16-15.35,32.52,32.52,0,0,1,18.28-5.59q15.39,0,24.42,10.51t9,28.71v5.24H780.46v1.79q0,14.39,8.24,24a26.21,26.21,0,0,0,20.74,9.57,30.47,30.47,0,0,0,13.24-2.73A29.39,29.39,0,0,0,833.11,139L839,143.4Q828.67,158.32,809.05,158.32Zm-1.72-79.69a23.36,23.36,0,0,0-17.77,7.74q-7.23,7.72-8.79,20.78h50.7v-1q-.39-12.18-7-19.84A21.55,21.55,0,0,0,807.33,78.63Z" transform="translate(-0.5 -0.5)" style="fill: rgb(255, 255, 255);"/>
+    <path d="M876,137.3l1.4,7.27,1.88-7.66L899.6,72.23h8l20.08,64.14,2.19,8.9,1.79-8.2L949,72.23h9.68l-24.61,84.53h-8l-21.72-67-1-4.61-1,4.68-21.33,67h-8L848.51,72.23h9.61Z" transform="translate(-0.5 -0.5)" style="fill: rgb(255, 255, 255);"/>
+    <path d="M1025,156.76a44.11,44.11,0,0,1-1.8-11.8,32.51,32.51,0,0,1-12.54,9.88,38.36,38.36,0,0,1-16.13,3.48q-12.19,0-19.73-6.8a22.15,22.15,0,0,1-7.54-17.18q0-12.34,10.28-19.54t28.63-7.18h17V98q0-9.06-5.58-14.26t-16.29-5.2q-9.76,0-16.17,5t-6.41,12l-9.38-.08q0-10.08,9.38-17.46t23-7.39q14.13,0,22.3,7.07t8.4,19.73v40q0,12.27,2.58,18.36v.94ZM995.58,150a31.4,31.4,0,0,0,16.76-4.53,26.68,26.68,0,0,0,10.74-12.11V114.8h-16.72q-14,.17-21.87,5.12t-7.89,13.63a15.13,15.13,0,0,0,5.27,11.8Q987.15,150,995.58,150Z" transform="translate(-0.5 -0.5)" style="fill: rgb(255, 255, 255);"/>
+    <path d="M1080.9,143.32l24.69-71.09h10.07l-36.09,98.9-1.87,4.38q-7,15.39-21.49,15.39a26.22,26.22,0,0,1-7.19-1.1l-.07-7.73,4.84.47q6.87,0,11.13-3.4t7.23-11.68l4.14-11.41-31.88-83.82h10.24Z" transform="translate(-0.5 -0.5)" style="fill: rgb(255, 255, 255);"/>
+  </g>
+  <g id="b074a1e9-6de5-4314-8d6e-719a56511290" data-name="D amrk">
+    <path d="M170.21,29.79A100,100,0,0,0,.5,86.36L43.64,43.22A80,80,0,0,1,156.78,156.36L113.64,199.5A100,100,0,0,0,170.21,29.79Z" transform="translate(-0.5 -0.5)" fill="#1e5df8"/>
+    <path d="M160.12,99.79A60,60,0,0,0,57.78,57.37L.5,114.65A100,100,0,0,0,6.39,137L71.92,71.51a40,40,0,0,1,56.57,56.57L63,193.61a100,100,0,0,0,22.39,5.89l57.28-57.28A59.57,59.57,0,0,0,160.12,99.79Z" transform="translate(-0.5 -0.5)" fill="#1e5df8"/>
+    <path d="M114.35,85.65a20,20,0,0,0-28.29,0l-70,70a101,101,0,0,0,28.29,28.29l70-70A20,20,0,0,0,114.35,85.65Z" transform="translate(-0.5 -0.5)" fill="#1e5df8"/>
+  </g>
+</svg>
```

Comment:
This is a new image that is the same as the standard DataGateway logo but with white text. The one with black text wasn't easily visible against the background image.

Reply:
This image is not actually needed because a logo with white text already existed! Will remove in a later commit.